src/coreclr/nativeaot/System.Private.StackTraceMetadata/src/Internal/StackTraceMetadata/MethodNameFormatter.cs

```diff
@@ -407,6 +411,21 @@ private void EmitPointerTypeName(PointerSignatureHandle pointerSigHandle)
             _outputBuilder.Append('*');
         }
 
+        /// <summary>
+        /// Emit function pointer type.
+        /// </summary>
+        /// <param name="functionPointerSigHandle">Function pointer type specification signature handle</param>
+        private void EmitFunctionPointerTypeName(FunctionPointerSignatureHandle functionPointerSigHandle)
+        {
+            FunctionPointerSignature functionPointerSig = _metadataReader.GetFunctionPointerSignature(functionPointerSigHandle);
+            _outputBuilder.Append("delegate*<");
```

Comment:
This should emit whatever CoreCLR will emit for this, which is probably not delegate*. Not at my PC anymore but I'm guessing IntPtr. Thanks for looking into this!